src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -67,15 +67,26 @@ def generate_mobile_frame(parameters: Dict[str, Optional[str]]) -> Dict[str, str
     return frame
 
 
-def set_top_tags(scope: Scope, project: Project) -> None:
-    scope.set_tag("project.slug", project.slug)
-    scope.set_tag("organization.slug", project.organization.slug)
+def set_top_tags(
+    scope: Scope,
+    project: Project,
+    ctx: Mapping[str, Optional[str]],
+    has_code_mappings: bool,
+) -> None:
     try:
-        ea_org: bool = project.organization.flags.early_adopter.is_set
-        scope.set_tag("organization.early_adopter", ea_org)
+        scope.set_tag("project.slug", project.slug)
+        scope.set_tag("organization.slug", project.organization.slug)
+        scope.set_tag(
+            "organization.early_adopter", bool(project.organization.flags.early_adopter.is_set)
+        )
+        scope.set_tag("stacktrace_link.platform", ctx["platform"])
+        scope.set_tag("stacktrace_link.has_code_mappings", has_code_mappings)
```

Comment:
This is new and it will help us distinguish requests for projects that have code mappings from those that don't.